network-api/networkapi/templates/fragments/buyersguide/pni_category_nav.html

```diff
@@ -0,0 +1,39 @@
+{% load bg_nav_tags localization i18n static wagtailroutablepage_tags wagtailmetadata_tags %}
+<nav id="multipage-nav" class="pni-category-nav text-center d-none d-md-block tw-no-scrollbar tw-mt-2" title="{% trans "site navigation" context "Tooltip on menu items" %}">
+    <div class="container tw-py-2" id="product-review">
+      <div class="row">
+        <div class="col">
+          <div id="pni-category-wrapper" class="tw-flex tw-items-center tw-w-max tw-min-w-full">
+            {% if pagetype == "product" or pagetype == "about" %}
+              <a class="multipage-link tw-block active" data-name="None" data-mobile="{% trans "All Categories" %}" href="{% relocalized_url home_page.localized.url %}">{% trans "All Reviews" %}</a>
```

Comment:
```suggestion
              <a class="multipage-link tw-block active" data-name="None" data-mobile="{% trans "All Categories" %}" href="{% relocalized_url home_page.localized.url %}">{% trans "All Product Reviews" %}</a>
```

Hey Simon! Just wanted to flag that in the original tickets AC I can see a item:
 - [ ] Category header has correct - "All Product Reviews" on homepage

I am not sure if having it say "All Reviews" is intentional, so I am leaving this suggestion just in case 👍